src/templates/Gallery.jsx

```diff
@@ -0,0 +1,244 @@
+            <Col className="d-flex justify-content-end">
+              <button
+                type="button"
+                className="btn btn-lg btn-primary rounded-circle media-button-cancel"
+                onClick={hideImage}
+              >
+                <IconContext.Provider value={closeIconAttributes}>
+                  <ImCross />
+                </IconContext.Provider>
+              </button>
+            </Col>
+            <Col className="d-flex justify-content-center">
+              <Carousel
+                fade
+                slide={false}
+                activeIndex={activeIndex}
+                next={showNext}
+                prev={showPrev}
+              >
+                <CarouselIndicators
+                  items={imageList}
+                  activeIndex={activeIndex}
+                  onClickHandler={goToIndex}
+                />
+                {slides}
+                <CarouselControl
+                  direction="prev"
+                  directionText="<"
+                  onClickHandler={showPrev}
+                />
+                <CarouselControl
+                  direction="next"
+                  directionText=">"
+                  onClickHandler={showNext}
+                />
+              </Carousel>
+            </Col>
+          </div>
+        </div>
+      ) : (
+        ""
+      )}
+      {/* Image Grid */}
+      <Col lg="12" className="card-spacing">
+        <Row className="justify-content-center">
+          {imageList.map(({ image }, index) => (
+            <Col className="pt-0 mt-2 mb-4" lg="4" md="8" sm="12">
+              <Card
+                className="row_shadow h-100 align-items-center"
+                onClick={() => {
+                  setImageDisplay(true);
+                  setActiveIndex(index);
+                  navigate(
+                    galleryCurrentPage === 1
+                      ? `${prefix}?index=${index}`
+                      : `${prefix}/${galleryCurrentPage}?index=${index}`
+                  );
+                }}
+                style={{ borderColor: "#08a045" }}
+              >
+                <GatsbyImage className="h-100" image={image} />
+              </Card>
+            </Col>
+          ))}
+        </Row>
+      </Col>
+      {/* Pagination */}
+      <div className="d-flex page-section">
+        {sildeNumber.map((number) => (
+          <PaginationItem key={number} className="mt-3">
+            <PaginationLink
+              href={number === 0 ? `${prefix}` : `${prefix}/${number + 1}`}
+            >
+              <IconContext.Provider value={paginationAttributes}>
+                {(() => {
+                  if (galleryCurrentPage === number + 1) {
+                    return <GiPlainSquare />;
+                  }
+                  return <GiSquare />;
+                })()}
+              </IconContext.Provider>
+            </PaginationLink>
+          </PaginationItem>
+        ))}
+      </div>
```

Comment:
Follow-up: ensure we use the same pagination principles on our pages (why do we have "previous page" / "next page" for modules and blogs, but use filled and non-filled icons for the gallery. This could hopefully become a reasonable reuse component as well to just add pagination to a page.

Reply:
doesn't that conflict with your idea of paginating modules by first letter? that's nothing we would do for blogs or gallery images, would we?

Reply:
:thinking: true... but that leaves us in an even worse spot, using a different pagination bar each time... :see_no_evil: maybe we call it a _design element_ :zany_face: